<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DiveLogImportDialog</class>
 <widget class="QDialog" name="DiveLogImportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>522</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Import dive log file</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/subsurface-icon</normalon>
   </iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>CSV options</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0" rowspan="3">
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Field configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="3" column="1">
           <widget class="QSpinBox" name="CSVpo2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="CSVDepth">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="CSVTemperature">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Depth</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QSpinBox" name="CSVstopdepth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QCheckBox" name="stopdepthCheckBox">
            <property name="text">
             <string>Stopdepth</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QSpinBox" name="CSVpressure">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QCheckBox" name="pressureCheckBox">
            <property name="text">
             <string>Pressure</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="po2CheckBox">
            <property name="text">
             <string>PO2</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="temperatureCheckBox">
            <property name="text">
             <string>Temp</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="CSVTime">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="CSVcns">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="ndlCheckBox">
            <property name="text">
             <string>NDL</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="CSVndl">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QCheckBox" name="ttsCheckBox">
            <property name="text">
             <string>TTS</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QSpinBox" name="CSVtts">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="cnsCheckBox">
            <property name="text">
             <string>Cns</string>
            </property>
           </widget>
          </item>
         </layout>
         <zorder>ndlCheckBox</zorder>
         <zorder>CSVndl</zorder>
         <zorder>ttsCheckBox</zorder>
         <zorder>CSVtts</zorder>
         <zorder>label</zorder>
         <zorder>label_2</zorder>
         <zorder>CSVTime</zorder>
         <zorder>CSVDepth</zorder>
         <zorder>temperatureCheckBox</zorder>
         <zorder>CSVTemperature</zorder>
         <zorder>po2CheckBox</zorder>
         <zorder>CSVpo2</zorder>
         <zorder>cnsCheckBox</zorder>
         <zorder>CSVcns</zorder>
         <zorder>stopdepthCheckBox</zorder>
         <zorder>CSVstopdepth</zorder>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Field separator</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="CSVSeparator"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Pre-configured imports</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QComboBox" name="knownImports">
            <property name="currentIndex">
             <number>-1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Units</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4b">
          <item>
           <widget class="QComboBox" name="CSVUnits">
            <item>
             <property name="text">
              <string>Metric</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Imperial</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Manual dives</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Units</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QComboBox" name="Units">
            <item>
             <property name="text">
              <string>Metric</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Imperial</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0" rowspan="2">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Field configuration</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="0" column="0">
           <widget class="QCheckBox" name="DiveNumberCheck">
            <property name="text">
             <string>Dive #</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="DiveNumber">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QCheckBox" name="MaxDepthCheck">
            <property name="text">
             <string>Max depth</string>
            </property>
           </widget>
          </item>
          <item row="0" column="3">
           <widget class="QSpinBox" name="MaxDepth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="DateCheck">
            <property name="toolTip">
             <string>Date in dd.mm.yyyy, yyyy-mm-dd or mm/dd/yyyy format</string>
            </property>
            <property name="text">
             <string>Date</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="Date">
            <property name="toolTip">
             <string>Date in dd.mm.yyyy, yyyy-mm-dd or mm/dd/yyyy format</string>
            </property>
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QCheckBox" name="MeanDepthCheck">
            <property name="text">
             <string>Mean depth</string>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QSpinBox" name="MeanDepth">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="TimeCheck">
            <property name="text">
             <string>Time</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="Time">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QCheckBox" name="BuddyCheck">
            <property name="text">
             <string>Buddy</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QSpinBox" name="Buddy">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="DurationCheck">
            <property name="toolTip">
             <string>Dive duration in seconds or in min:sec format</string>
            </property>
            <property name="text">
             <string>Duration</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="Duration">
            <property name="toolTip">
             <string>Dive duration in seconds or in min:sec format</string>
            </property>
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QCheckBox" name="NotesCheck">
            <property name="text">
             <string>Notes</string>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QSpinBox" name="Notes">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="LocationCheck">
            <property name="text">
             <string>Location</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="Location">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QCheckBox" name="WeightCheck">
            <property name="text">
             <string>Weight</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QSpinBox" name="Weight">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QCheckBox" name="GpsCheck">
            <property name="text">
             <string>Gps</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="Gps">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QCheckBox" name="TagsCheck">
            <property name="text">
             <string>Tags</string>
            </property>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QSpinBox" name="Tags">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimum">
             <number>1</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Field separator</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QComboBox" name="ManualSeparator"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="horizontalWidget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="verticalWidget" native="true">
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QDialogButtonBox" name="buttonBox">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="standardButtons">
            <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>CSVTime</tabstop>
  <tabstop>CSVDepth</tabstop>
  <tabstop>temperatureCheckBox</tabstop>
  <tabstop>CSVTemperature</tabstop>
  <tabstop>po2CheckBox</tabstop>
  <tabstop>CSVpo2</tabstop>
  <tabstop>cnsCheckBox</tabstop>
  <tabstop>CSVcns</tabstop>
  <tabstop>stopdepthCheckBox</tabstop>
  <tabstop>CSVstopdepth</tabstop>
  <tabstop>CSVSeparator</tabstop>
  <tabstop>knownImports</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DiveLogImportDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DiveLogImportDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>467</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>164</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>temperatureCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>CSVTemperature</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>171</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>po2CheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>CSVpo2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cnsCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>CSVcns</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>233</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>235</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopdepthCheckBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>CSVstopdepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>108</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>238</x>
     <y>267</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DiveNumberCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>DiveNumber</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DateCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Date</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TimeCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Time</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>LocationCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Location</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>GpsCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Gps</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MaxDepthCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MaxDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MeanDepthCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>MeanDepth</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>137</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>BuddyCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Buddy</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>NotesCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Notes</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TagsCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Tags</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>263</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WeightCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Weight</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>185</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>233</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>DurationCheck</sender>
   <signal>clicked(bool)</signal>
   <receiver>Duration</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>96</x>
     <y>199</y>
    </hint>
    <hint type="destinationlabel">
     <x>176</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
